subscriptions/README.md

```diff
@@ -1,3 +1,3 @@
-# Subscriptions v1.0.0-wip
+# CNCF CloudEvents – Subscriptions API - Version 0.1-wip
```

Comment:
We should probably be consistent about whether there's a "CNCF" there or not, and whether "CloudEvents" is there too? E.g. "CNCF CloudEvents Subscription API" vs "CloudEvents Subscription API" vs "Subscription API". I'm leaning towards just "Subscription API" as it feels redundant to include the others, but any are ok with me. Perhaps we should discuss on the call?